activemount.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/docker/go-plugins-helpers/volume"
+)
+
+type ActiveMount struct {
+	UsageCount int
+}
+
+func (d *DockerOnTop) Activate(request *volume.MountRequest, activemountsdir lockedFile) (bool, error) {
+	var activeMount ActiveMount
+	var result bool
```

Comment:
Since this was an issue also in driver.go (see [comment](https://github.com/kolayne/docker-on-top/pull/20/files/bb0d01096262eea9b5f04e53851963f7f45fe453#r1694002909)), I have named it the same way everywhere: 1985b5201fee166a4fba0d11e1a8c1026c247f02